### DISCUSSION TITLE
Masque not Saving Settings

### DISCUSSION THREAD

**Discussion by siven80** [SFX-WoW/Masque#182] [Issues] [answered]
After setting adibuttonsauras to use a masque skin (tried with simplesquares and Caith) it works fine, changing the border to green for example after using Battle shout.

When i relog it seems to forget the skin and just uses the blizzard skin where the edge just sort of glows.

It seem to be the only addon the does this and the buffs and Domino bars keep their settings.

Not sure of the issue is with this or adibuttonauras so posted on both.

**Comment by StormFX** [comment]
Sounds like an issue with ABA overriding Masque's settings. If it was an issue with Masque, it wouldn't work on _any_ add-ons.

**Comment by BalurPoco** [comment]
WeakAuras has the same problem. I set it correctly and after a combat or two its go back to Blizzard skin.
https://github.com/WeakAuras/WeakAuras2/issues/639

**Comment by siven80** [comment, marked as answer]
Found out it was a prob;em with dominos as bartender had no problems.
